omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_indexing_validator.rb

```diff
@@ -117,7 +117,9 @@ def elasticsearch_enabled?
   def warn_unchanged_external_flag
     if OmnibusHelper.has_been_bootstrapped? && backend? && previous_run
       # ES configuration is preferred everywhere so we check that first
-      previous_external_setting = if previous_run['elasticsearch'].key?('external')
+      # Check if the elasticsearch hash wiill not be present for versioons before 13.2.0
```

Comment:
```suggestion
      # Check if the elasticsearch hash will not be present for versioons before 13.2.0
```

Reply:
I'm wondering...is this check even worth it?

Reply:
Hrmm.. This check was useful when we were going to include solr as an option in 14.x. Since that is removed:
13 (internal_solr) -> 14 elasticsearch : reindex with be run on upgrade
13 (snowflake)    -> 14 elasticsearch : skip reindex on upgrade
13 (external_es) -> 14 (we do not manage that)

So I can remove this completely! Nice observation.